gcc/config.gcc

```diff
@@ -1264,24 +1264,40 @@ aarch64*-*-linux*)
 	TM_MULTILIB_CONFIG=`echo $TM_MULTILIB_CONFIG | sed 's/^,//'`
 	;;
 aarch64*-*-mingw*)
-# 	tm_file="${tm_file} dbxelf.h elfos.h gnu-user.h linux.h glibc-stdint.h"
-# 	tm_file="${tm_file} aarch64/aarch64-elf.h aarch64/aarch64-errata.h aarch64/aarch64-linux.h"
-# 	tm_file="${tm_file} i386/unix.h i386/bsd.h i386/gas.h dbxcoff.h i386/cygming.h"	
-	tm_file="${tm_file} aarch64/aarch64-coff.h"
-	tm_file="${tm_file} aarch64/cygming.h"
+	tm_file="${tm_file} aarch64/aarch64-coff.h aarch64/cygming.h" # i386/unix.h i386/bsd.h i386/gas.h
```

Comment:
I would exclude comment from refactoring

Reply:
They are there on purpose of noting that those files are sort of TODO. Should I make them more explicit, e.g.:
```
# TODO: aarch64-linux-gnu target imports i386/unix.h, i386/bsd.h, i386/gas.h. Investigate whether those are relevant also for aarch64-w64-mingw32 target?
``